business/products.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Providers of machine translation services
+---
+
+# Providers
+
+Machine translation **providers** offer generic machine translation services with different customization possibilities. Machine translation services have different features and limitations.
+
+The advantage of using a machine translation provider is that the service is already working and can be implemented immediately. The alternative is to train a machine translation engine from scratch.
```

Comment:
"integrated", not "implemented"

"The alternative is to train a model from scratch or to use a pretrained model, and deploy and maintain a machine translation system."